composer.json

```diff
@@ -20,14 +20,17 @@
   ],
   "license": "MIT",
   "require": {
-    "php": ">=5.3.9",
-    "apache/log4php": "2.3.*",
-    "symfony/validator": "2.7.*",
-    "doctrine/annotations":"1.2.*",
-    "doctrine/cache":"1.4.*"
+    "php": "^7.1",
+    "apache/log4php": "2.3.0",
+    "symfony/validator": "^2.7|^3.0",
```

Comment:
Needed to allow upgrading in other projects (suntransfers.com and admin.suntransfers.com for instance) to Symfony 3.x